### ISSUE TITLE
Nessus finding for CVE-2022-22719

### ISSUE THREAD

**Issue by phydroxide** [docker-library/php#1276]
https://github.com/docker-library/php/blob/f2754514dafb2fa33d1b4cd7a152a35bacebf95f/7.4/bullseye/apache/Dockerfile#L56

Nessus finds vulnerabilities in wordpress:latest container as of 3/23. This depends on php7.4 container
March 14:
https://nvd.nist.gov/vuln/detail/CVE-2022-22719
https://cve.mitre.org/cgi-bin/cvename.cgi?name=2022-22719

Description
The version of Apache httpd installed on the remote host is prior to 2.4.53. It is, therefore, affected by multiple vulnerabilities as referenced in the 2.4.53 advisory.

- mod_lua Use of uninitialized value of in r:parsebodyA carefully crafted request body can cause a read to a random memory area which could cause the process to crash. This issue affects Apache HTTP Server 2.4.52 and earlier. Acknowledgements: Chamal De Silva (CVE-2022-22719)

- HTTP request smuggling vulnerability in Apache HTTP Server 2.4.52 and earlierApache HTTP Server 2.4.52 and earlier fails to close inbound connection when errors are encountered discarding the request body, exposing the server to HTTP Request Smuggling Acknowledgements: James Kettle <james.kettle portswigger.net> (CVE-2022-22720)

- core: Possible buffer overflow with very large or unlimited LimitXMLRequestBodyIf LimitXMLRequestBody is set to allow request bodies larger than 350MB (defaults to 1M) on 32 bit systems an integer overflow happens which later causes out of bounds writes. This issue affects Apache HTTP Server 2.4.52 and earlier. Acknowledgements: Anonymous working with Trend Micro Zero Day Initiative (CVE-2022-22721)

- mod_sed: Read/write beyond boundsOut-of-bounds Write vulnerability in mod_sed of Apache HTTP Server allows an attacker to overwrite heap memory with possibly attacker provided data. This issue affects Apache HTTP Server 2.4 version 2.4.52 and prior versions. Acknowledgements: Ronald Crane (Zippenhop LLC) (CVE-2022-23943)

Note that Nessus has not tested for this issue but has instead relied only on the application's self-reported version number.
Solution
Upgrade to Apache version 2.4.53 or later.

**Comment by tianon** [comment]
Duplicate of https://github.com/docker-library/php/issues/1273
